### DISCUSSION TITLE
Relation to ElevenLabs?

### DISCUSSION THREAD

**Discussion by catboxanon** [neonbjb/tortoise-tts#277] [General]
Does anyone know if this project has any relation to ElevenLabs? Their product touts a couple of the same features as this project does, although it may just be coincidence.
https://beta.elevenlabs.io/

>  Our cloning model learns any speech profile based on just a minute of audio, without training.

> [...] lets you create completely new voices that never spoke before.

**Comment by vdruts** [comment]
I'm almost positive its the same underlying tech that they took (learned how to train) and fine-tune.

**Comment by protomato2** [comment]
I been wondering about that. I get there is no affiliation, but  I wonder if it works by the same principle but with larger scale model

**Comment by catboxanon** [comment]
Leaving this here: https://github.com/neonbjb/tortoise-tts/discussions/292#discussioncomment-4947111

**Comment by catboxanon** [comment]
> 11labs **is** Tortoise

https://github.com/neonbjb/tortoise-tts/discussions/292#discussioncomment-5217149

Probably take with a large grain of salt.
